### ISSUE TITLE
WebRtcVad - DllNotFoundException

### ISSUE THREAD

**Issue by wCATw** [qwbarch/mirage#77]
*I use translator.

My friend hears me, but I can't hear him. We tried to reconnect the microphones, connect to the lobby, restart the game. Nothing is working. The game is running as an administrator. BarchLib is the latest version. All mods are installed manually. Virality is installed, but only one person has it, and others communicate freely.
![Log](https://github.com/user-attachments/assets/f2849357-3df6-42b5-bfb5-7036b120481d)

**Comment by wCATw** [comment]
![mods list](https://github.com/user-attachments/assets/c6b55f3d-15e6-4c10-957e-a9318cbf82d8)
We have these mods (except MoreComments and Mirage itself)

**Comment by wCATw** [comment]
And this in core folder
![core folder](https://github.com/user-attachments/assets/ca6bfbe4-2942-4da0-a3a5-0711c9c47f94)

**Comment by qwbarch** [comment]
I'm not sure why it's not able to find the dll when your screenshot shows it. Could you try using a mod manager (like r2modman) and see if it works for you?
If it does, I'd recommend just switching over to it

**Comment by wCATw** [comment]
Okay, I'll try to use the mod managers.

**Comment by wCATw** [comment]
Can there be any dependencies missing, for example, a certain version .NET? FSharp or Photon packages? The friend who had this problem, recently forcibly reset Windows due to a critical error.

**Comment by qwbarch** [comment]
It shouldn't need to. As far as I'm aware, all the dependencies it needs are bundled into BarchLib itself.
Guessing using a modmanager didn't change anything?

**Comment by wCATw** [comment]
> Guessing using a modmanager didn't change anything?

Yeah. I tried to buy all my friends a copy of the game (one of my friends didn't have it and we played with a SteamFix), but that didn't help either.

**Comment by wCATw** [comment]
This is a full log
[LogOutput.log](https://github.com/user-attachments/files/16573469/LogOutput.log)

**Comment by wCATw** [comment]
And order of loading mods
![order](https://github.com/user-attachments/assets/c6fa6d5f-c027-41bf-8c86-7cca4907372e)

**Comment by wCATw** [comment]
Oh shit. Visual C++ was deleted from a friend due to reinstalling Windows. After he installed it, everything worked. Specify Visual C++ dependencies. Thanks for trying to help. I'm closing the issue.
@qwbarch

**Comment by qwbarch** [comment]
I always assumed my dll bundled that, but good to know. Thanks for mentioning that, I'll point that to anyone else that runs into this in the future
